components/SiteNavbar.vue

```diff
@@ -1,38 +1,231 @@
+            font-semibold
+            leading-6
+            tracking-tight
+            lg:text-left
+          "
+        >
+          speedrun.website
+        </h1>
+      </div>
+      <div class="flex">
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          class="h-10 w-10 mx-1 lg:hidden"
+          fill="none"
+          viewBox="0 0 24 24"
+          stroke="currentColor"
+        >
+          <path
+            stroke-linecap="round"
+            stroke-linejoin="round"
+            stroke-width="2"
+            d="M12 8v4l3 3m6-3a9 9 0 11-18 0 9 9 0 0118 0z"
+          />
         </svg>
-      </button>
-    </div>    
-    <div class="flex flex-grow">
-      <h1 class="w-full py-2 text-lg text-center font-semibold leading-6 tracking-tight lg:text-left">speedrun.website</h1>
+      </div>
+      <div
+        class="
+          flex flex-col
+          w-full
+          my-2
+          text-center
+          font-semibold
+          lg:flex-row lg:w-auto lg:my-0
+        "
+        v-show="mobileNavIsActive || lg"
+      >
+        <a
+          class="
+            rounded
+            px-3
+            py-2
+            m-1
+            text-gray-600
+            lg:my-0
+            hover:bg-gray-100 hover:text-gray-800
+          "
+          href="#"
+          >Home</a
+        >
+        <a
+          class="
+            rounded
+            px-3
+            py-2
+            m-1
+            text-gray-600
+            lg:my-0
+            hover:bg-gray-100 hover:text-gray-800
+          "
+          href="#"
+          >Games</a
+        >
+        <a
+          class="
+            rounded
+            px-3
+            py-2
+            m-1
+            text-gray-600
+            lg:my-0
+            hover:bg-gray-100 hover:text-gray-800
+          "
+          href="#"
+          >About</a
+        >
+      </div>
+      <div
+        class="
+          relative
+          flex flex-col
+          w-full
+          px-2
+          my-1
+          lg:flex-row lg:w-auto lg:my-0
+        "
+      >
+        <input
+          class="
+            border border-gray-400
+            rounded
+            bg-white
+            px-4
+            py-2
+            pr-12
+            focus:outline-none
+          "
+          name="search"
+          placeholder="Search..."
+        />
+        <button type="submit" class="absolute right-6 bottom-3">
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            class="h-5 w-5"
+            fill="none"
+            viewBox="0 0 24 24"
+            stroke="currentColor"
+          >
+            <path
+              stroke-linecap="round"
+              stroke-linejoin="round"
+              stroke-width="2"
+              d="M21 21l-6-6m2-5a7 7 0 11-14 0 7 7 0 0114 0z"
+            />
+          </svg>
+        </button>
+      </div>
+      <div
+        class="
+          flex flex-col
+          w-full
+          my-2
```

Comment:
Could the `my-2` be removed? It creates a lot of white space in the hamburger menu in the mobile layout